tests/test_bad.py

```diff
@@ -0,0 +1,136 @@
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import os
+import shutil
+import tempfile
+
+import asynctest
+import lsst.utils.tests
+import yaml
+from lsst.ctrl.oods.directoryScanner import DirectoryScanner
+from lsst.ctrl.oods.fileIngester import FileIngester
+from lsst.ctrl.oods.utils import Utils
+
+
+class BadTestCase(asynctest.TestCase):
+    """Test Gen3 Butler Ingest"""
+
+    def createConfig(self, config_name, fits_name):
```

Comment:
This is a lot of boilerplate.  You may want to create your own subclass that packages all of this, as I think I see it being used in other tests as well.

Reply:
Yes, I'll put in a ticket to refactor for this and the suggestion below.